frontend/Dockerfile

```
@@ -0,0 +1,17 @@
+# syntax=docker/dockerfile:1.5
+
+FROM node:22-bullseye
+WORKDIR /app
+
+ENV NODE_ENV=development
+ENV VITE_SIGNALING_WS_URL=ws://localhost:8080/ws
+
+# Install dependencies using cached layers when package manifests do not change
+COPY package.json package-lock.json* .
```

Hadolint (2.14.0)

[error] COPY with more than 2 arguments requires the last argument to end with /

(DL3021)